packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -71,6 +71,17 @@ const actualPopperClass = computed(() => {
     result.push('no-arrow')
   }
 
+  // color takes priority, else fallback to tooltip -> dark, menu -> light
+  if (props.color === 'dark') {
```

Comment:
Can you add some scenarios to the component test for this to show off these new options?

Reply:
Added tests in [52feb4b](https://github.com/cypress-io/cypress/pull/25234/commits/52feb4bb9e19e34910da6de2cc730478aa1cfb68)